e2e/test/netstandard20/ProvisioningE2ETests.cs

```diff
@@ -263,6 +263,84 @@ public async Task ProvisioningDeviceClient_ValidRegistrationId_MqttWs_SymmetricK
             await ProvisioningDeviceClient_ValidRegistrationId_Register_Ok(Client.TransportType.Mqtt, AttestationType.SymmetricKey, EnrollmentType.Individual, false).ConfigureAwait(false);
         }
 
+        [TestMethod]
```

Comment:
Please write another test - this is already getting too crowded.
Also, please vary only relevant items - I don't see how using TPM vs X509 would be relevant for how we fill in `DeviceCapabilities()`. I think having 3 tests for the 3 protocols should be sufficient (WSS won't matter here either).

Reply:
Protocol shouldn't matter, nor should attestation type, you are right. Individual enrollment vs group enrollment does matter, so I'll just have one test case for each of those instead of what I have right now.

Reply:
As for the test suite getting too large, I'll take some time to move all of the reprovisioning tests into a separate suite for this PR. The tests I'm writing for device capabilities do belong in this suite, though.